### PR TITLE
added sentence to explain when a PR will be merged

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ To manage changes, we use [GitHub flow](https://guides.github.com/introduction/f
 
 For beginners, the GitHub interface will help you in the process of editing a file. It will automatically create a fork of this repository where you can safely work and then submit the changes as a pull request without having to touch the command line.
 
+Pull requests will be merged by the [core team](https://github.com/orgs/galaxyproject/teams/training/members) after at least one other person has reviewed the Pull request and approved it.
```
Comment (willdurand) [inline CONTRIBUTING.md]: 2 quick points:

- the link does not work if you're not an admin/privileged user of the galaxyproject org (for instance, I dont have access to it)
- is that a new rule regarding the training-material project? It seems I had merge rights until now and I dont think I am a "core team" member.

Comment (bgruening) [reply CONTRIBUTING.md]: Obviously visible means only visible to a some and not all: "A visible team can be seen and @mentioned by every member of this organization." I removed the link.

Comment (willdurand) [reply CONTRIBUTING.md]: Yeah this has always been weird to me..

**Comment by bgruening** [comment]
@nekrut better?
```
Pull requests will be merged by the training team members after at least one reviewer (training team member) has approved the Pull request.
```

**Pull request merged**